usaspending_api/broker/management/commands/load_fpds_transactions.py

```diff
@@ -151,8 +151,9 @@ def handle(self, *args, **options):
             self.load_fpds_incrementally(last_load)
 
         if self.modified_award_ids:
-            logger.info("cleaning orphaned metadata")
-            destroy_orphans()
+            # TODO: reactivate once this is performant. Currently the tables are too large to allow
+            # logger.info("cleaning orphaned metadata")
```

Comment:
@BrianZito - I think this would have been a good place for `logger.warning("WARNING: Cleanup of orphaned metadata is being skipped")`